src/cursor/cur_metadata.c

```diff
@@ -79,14 +80,17 @@ __schema_create_collapse(WT_SESSION_IMPL *session, WT_CURSOR_METADATA *mdc, cons
     c = NULL;
     if (key != NULL && WT_PREFIX_SKIP(key, "table:")) {
         /*
-         * Check if the table has declared column groups. If it does, don't attempt to open the
-         * automatically created column group for simple tables.
+         * Check if the table has declared column groups. If it does, return ENOTSUP. One can get
+         * the creation metadata for an index or column group table itself, but it is not defined
+         * what that is for a complex table URI.
```

Comment:
It looks like the `ENOTSUP` part of the comment is no longer relevant.

Reply:
I would've sworn I changed that, but I must be thinking of some other location. In any case, fixed now.